### PR TITLE
Fix antiaim

### PR DESCRIPTION
**Pull request by fartdev**
Freelook with antiaim disabled remains unchanged but this pr fixes the freelook fix, so now you can walk around with antiaim on

### CODE REVIEW

File: src/main/java/coffee/client/mixin/entity/LivingEntityMixin.java [hunk 1]
```diff
@@ -69,7 +69,7 @@ public void coffee_cancelCollision(Entity entity, CallbackInfo ci) {
 
     @Redirect(method = "jump", at = @At(value = "INVOKE", target = "net/minecraft/entity/LivingEntity.getYaw()F"), require = 0)
     private float coffee_overwriteJumpDirection(LivingEntity instance) {
-        if (instance.equals(CoffeeMain.client.player) && ModuleRegistry.getByClass(FreeLook.class).isEnabled()) {
+        if (instance.equals(CoffeeMain.client.player) && ModuleRegistry.getByClass(FreeLook.class).isEnabled() && !((boolean) ModuleRegistry.getByClass(FreeLook.class).config.get("Enable Anti-Aim").getValue())) {
```
Comment (0x3C50) [inline src/main/java/coffee/client/mixin/entity/LivingEntityMixin.java]: Change to getter for direct reference, getting by name is wonky as shit

File: src/main/java/coffee/client/mixin/render/GameRendererMixin.java [hunk 2]
```diff
@@ -92,7 +92,7 @@ private void coffee_stopCursorBob(MatrixStack matrices, float f, CallbackInfo ci
 
     @Redirect(at = @At(value = "INVOKE", target = "Lnet/minecraft/entity/Entity;raycast(DFZ)Lnet/minecraft/util/hit/HitResult;"), method = "updateTargetedEntity", require = 0)
     HitResult coffee_replaceHitResult(Entity instance, double maxDistance, float tickDelta, boolean includeFluids) {
-        if (ModuleRegistry.getByClass(FreeLook.class).isEnabled()) {
+        if (ModuleRegistry.getByClass(FreeLook.class).isEnabled() && !((boolean) ModuleRegistry.getByClass(FreeLook.class).config.get("Enable Anti-Aim").getValue())) {
```
Comment (0x3C50) [inline src/main/java/coffee/client/mixin/render/GameRendererMixin.java]: See above

**Review by 0x3C50 — CHANGES_REQUESTED** [review]
Good commit but needs changes

**Comment by fartdev** [comment]
Should be better now, I used the same method you used for AntiCrash

**Review by 0x3C50 — APPROVED** [review]

**Pull request merged**